lib/definitions/preview-app-livesync.d.ts

```diff
@@ -34,4 +33,11 @@ declare global {
 		 */
 		link: boolean;
 	}
+
+	interface IPreviewDevicesService extends EventEmitter {
+		connectedDevices: Device[];
+		onDevicesPresence(devices: Device[]): void;
```

Comment:
In the context of this service, this should be something like set/update ConnectedDevices.